bin/validate.py

```diff
@@ -150,6 +151,7 @@ def run(self, testcase, run=None, constraints=None, args=None):
                     expect=config.RTV_WA if testcase.bad_output else config.RTV_AC,
                     stdin=ans_file,
                     cwd=cwd,
+                    timeout=self.problem.settings.timeout,
```

Comment:
I think it's better to read `config.args.timeout` directly. If you do `bt run --timelimit 2`, this will automatically set a timeout of `4` for submissions in `problem.settings.timeout`, which should not apply to validators.

Similar to https://github.com/RagnarGrootKoerkamp/BAPCtools/blob/master/bin/program.py#L467 for generators.

Coul you also extract the `30` here and elsewhere (also in `utils.py`) into a flag in `config.py`?

Reply:
I hope I found all :D